src/views/search/search-page.tsx

```diff
@@ -0,0 +1,99 @@
+import { useCallback, useMemo } from "preact/hooks";
+import { Pagination } from "components/pagination/pagination";
+import { PostInfo } from "types/PostInfo";
+import { PostCard } from "components/post-card/post-card";
+import unicornProfilePicMap from "../../../public/unicorn-profile-pic-map";
```

Comment:
This file doesn't exist on dev builds, so it would be preferred to use `utils/get-unicorn-profile-pic-map` in an astro file and pass this in as a prop